bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HBaseImportPipeline.java

```diff
@@ -0,0 +1,63 @@
+package com.google.cloud.bigtable.dataflow;
+
+import com.google.bigtable.repackaged.com.google.common.collect.ImmutableMap;
+import com.google.cloud.dataflow.sdk.Pipeline;
+import com.google.cloud.dataflow.sdk.coders.KvCoder;
+import com.google.cloud.dataflow.sdk.io.Read;
+import com.google.cloud.dataflow.sdk.transforms.ParDo;
+
+import org.apache.hadoop.hbase.client.Result;
+import org.apache.hadoop.hbase.io.ImmutableBytesWritable;
+import org.apache.hadoop.mapreduce.lib.input.SequenceFileInputFormat;
+
+/**
+ * Sets up and runs a dataflow pipeline to import HBase sequence file(s) into Cloud Bigtable.
+ *
+ * <p>User of this class should provide dependency for
+ * {@code org.apache.hadoop.hbase.mapreduce.ResultSerialization}.
+ */
+public class HBaseImportPipeline {
+
+  // Copied from ResultSerialization.IMPORT_FORMAT_VER
+  private static final String IMPORT_FORMAT_VER = "hbase.import.version";
+  private static final String VERSION_094_STRING = "0.94";
+
+  private final HBaseImportOptions options;
+
+  public HBaseImportPipeline(HBaseImportOptions options) {
+    this.options = options;
+  }
+
+  public void run() {
+    Pipeline p = Pipeline.create(options);
+    CloudBigtableIO.initializeForWrite(p);
+
+    p
+        .apply("ReadSequenceFile", Read.from(createFileSource(
+            options.getFilePattern(), options.getHBase094DataFormatOverride())))
+        .apply(ParDo.named("ResultToMutation").of(new HBaseResultToMutationFn()))
+        .apply(CloudBigtableIO.writeToTable(
+            CloudBigtableTableConfiguration.fromCBTOptions(options)));
+    p.run();
+  }
+
+  public static HadoopFileSource<ImmutableBytesWritable, Result> createFileSource(
```

Comment:
Can this be moved to CloudBigtableIO?  This could be useful for other pipelines.